src/notation/qml/MuseScore/NotationScene/PercussionPanelPadSwapDialog.qml

```diff
@@ -0,0 +1,236 @@
+            spacing: 16
+
+            StyledTextLabel {
+                id: titleLabel
+
+                width: parent.width
+
+                font: ui.theme.largeBodyBoldFont
+                horizontalAlignment: Text.AlignLeft
+                wrapMode: Text.Wrap
+
+                text: qsTrc("notation", "Do you also want to move the MIDI notes and keyboard shortcuts that trigger these sounds?")
+            }
+
+            RadioButtonGroup {
+                id: radioButtons
+
+                width: parent.width
+                spacing: contentColumn.spacing
+
+                orientation: ListView.Vertical
+
+                model: [
+                    { text: qsTrc("notation", "Move MIDI notes and keyboard shortcuts with their sounds"), value: true },
+                    { text: qsTrc("notation", "Leave MIDI notes and keyboard shortcuts fixed to original pad positions"), value: false }
+                ]
+
+                delegate: Row {
+                    width: parent.width
+                    spacing: 6
+
+                    RoundedRadioButton {
+                        id: radioButton
+
+                        anchors.verticalCenter: parent.verticalCenter
+
+                        navigation.name: modelData.text
+                        navigation.panel: contentNavigationPanel
+                        navigation.row: model.index
+
+                        checked: modelData.value === prv.moveMidiNotesAndShortcuts
+
+                        onToggled: {
+                            // TODO: Live update the pads when this value is changed...
+                            prv.moveMidiNotesAndShortcuts = modelData.value
+                        }
+                    }
+
+                    //! NOTE: Can't use radioButton.text because it won't wrap
+                    StyledTextLabel {
+                        width: parent.width - parent.spacing - radioButton.width
+
+                        anchors.verticalCenter: parent.verticalCenter
+
+                        horizontalAlignment: Text.AlignLeft
+                        wrapMode: Text.Wrap
+                        text: modelData.text
+
+                        MouseArea {
+                            id: mouseArea
+
+                            anchors.fill: parent
+
+                            onClicked: {
+                                // TODO: Live update the pads when this value is changed...
+                                prv.moveMidiNotesAndShortcuts = modelData.value
+                            }
+                        }
+                    }
+                }
+            }
+
+            CheckBox {
+                id: rememberMyChoice
+
+                width: parent.width
+
+                navigation.panel: contentNavigationPanel
+                navigation.row: radioButtons.model.length
+                navigation.accessible.name: rememberMyChoice.text
+
+                text: qsTrc("notation", "Remember my choice")
```

Comment:
The context should be "global". We should also change it for the same string in AlsoShareAudioComDialog.qml